libnavigation-util/src/main/java/com/mapbox/navigation/utils/extensions/HandlerThreadEx.kt

```diff
@@ -2,13 +2,9 @@
 
 package com.mapbox.navigation.utils.extensions
 
-import android.os.Build
 import android.os.HandlerThread
 
+@Deprecated("use HandlerThread.quitSafely() instead", ReplaceWith("this.quitSafely()"))
```

Comment:
Do we need this? I believe is not necessary.

Reply:
done